docs/data-types/streams.md

```diff
@@ -114,26 +118,26 @@ The format of such IDs may look strange at first, and the gentle reader may wond
 
 If for some reason the user needs incremental IDs that are not related to time but are actually associated to another external system ID, as previously mentioned, the `XADD` command can take an explicit ID instead of the `*` wildcard ID that triggers auto-generation, like in the following examples:
 
-```
-> XADD somestream 0-1 field value
+{{< clients-example stream_toturial xadd_id >}}
```

Comment:
I imagine this mispelling will likely trip you up at some point with the other client examples. There are several instances of this that need to be corrected.

```suggestion
{{< clients-example stream_tutorial xadd_id >}}
```

Reply:
lol yikes. fixed in the incoming commit